static/src/javascripts/projects/common/modules/spacefinder.js

```diff
@@ -1,18 +1,12 @@
 // total_hours_spent_maintaining_this = 72
 
-import qwery from 'qwery';
 import bean from 'bean';
 import fastdom from '../../../lib/fastdom-promise';
 import mediator from '../../../lib/mediator';
 import { memoize } from 'lodash-es';
 
-
-
-
-
-
-
-
+const query = (selector, context) =>
+    [...(context ?? document).querySelectorAll(selector)];
```

Comment:
qute!

Looks [a lot like $$](https://github.com/guardian/frontend/blob/062507eed21dc403be557b2e919f9a7ccfbe92c9/static/src/javascripts/lib/%24%24.ts#L14), though…

Reply:
Valid point but I'm personally not too keen on $$ as it keeps an internal array of elements and having to call `.get()` each time to get elements back out.
Also most places where we replaced qwery it didn't use context so a straight swap for querySelectorAll was less code and more explicit.
This was one of few places where context and array filtering was used so ended up replicating the selector part.